app/src/resolvers.ts

```diff
@@ -8,31 +8,49 @@ import { getJenkinsServerWithSecret } from './storage/get-jenkins-server-with-se
 import { updateJenkinsServer } from './storage/update-jenkins-server';
 import { deleteBuilds } from './jira-client/delete-builds';
 import { deleteDeployments } from './jira-client/delete-deployments';
+import { isAdmin } from './check-permissions';
 
 const resolver = new Resolver();
 
-resolver.define('fetchJenkinsEventHandlerUrl', async () => ({
-	url: await webTrigger.getUrl('jenkins-webtrigger')
-}));
+const checkPermissions = async (accountId: string) => {
```

Comment:
`checkPermissions` is not very explicit in terms of naming. It's not a general check for permissions, it's specifically to check if the user is an admin.

Reply:
Yeah that's fair. I was unsure what to name it as the function I import from the other file is `isAdmin`. Let me think about how to make this clearer.

Reply:
Updated name to `adminPermissionCheck` as discussed